app/src/main/java/com/zeapo/pwdstore/model/PasswordEntry.kt

```diff
@@ -31,9 +31,9 @@ class PasswordEntry(content: String, private val totpFinder: TotpFinder = UriTot
     constructor(os: ByteArrayOutputStream) : this(os.toString("UTF-8"), UriTotpFinder())
 
     init {
-        val passContent = content.split("\n".toRegex(), 2).toTypedArray()
-        password = if (UriTotpFinder.TOTP_FIELDS.any { passContent[0].startsWith(it) }) "" else passContent[0]
-        extraContent = findExtraContent(passContent)
+        val pair = findAndStripPassword(content.split("\n".toRegex()))
+        password = pair.first
+        extraContent = pair.second.joinToString("\n")
```

Comment:
I can change by the following
```suggestion
        val (foundPassword, passContent) = findAndStripPassword(content.split("\n".toRegex()))
        password = foundPassword
        extraContent = passContent.joinToString("\n")
```

Reply:
This looks good.